asapdiscovery-modeling/asapdiscovery/modeling/modeling.py

```diff
@@ -577,19 +577,24 @@ def split_openeye_mol(
         Molecule filter object that contains the filter criteria.
     prot_cutoff_len : int, default=10
         Minimum number of residues in a protein chain required in order to keep
+    keep_one_lig : bool, default=True
+        Only keep one copy of the ligand in lig_mol
 
     Returns
     -------
-    namedtuple
```

Comment:
Was the named tuple not working for the new design?

Reply:
previously only the protein molecule was getting returned, which is why I ended up reworking the function. I think `namedtuple` should be fine, I've just never used them before so I defaulted to `dict` lol. can switch it around if you think it would be better